test/nearest/ceil_test.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024-Present Ian Pike
+ * Copyright (c) 2024-Present ccmath contributors
+ *
+ * This library is provided under the MIT License.
+ * See LICENSE for more information.
+ */
+
+#include <gtest/gtest.h>
+
+#include <cmath>
+#include <limits>
+#include "ccmath/ccmath.hpp"
+
+TEST(CcmathNearestTests, Ceil)
+{
```

Comment:
We should also be checking that the function works with static_assert.

Reply:
how

Reply:
The simplest manner would be to just do as follows:
```cpp
static_assert(ccm::ceil(1.0) == ccm::ceil(1.0), “ccm::ceil is not static_assertable”);
```